bundles/org.openhab.binding.pulseaudio/README.md

```diff
@@ -18,6 +18,24 @@ You need a running pulseaudio server with module **module-cli-protocol-tcp** loa
 
 The Pulseaudio bridge is discovered through mDNS in the local network.
 
+## Binding Configuration (optional)
+
+The Pulseaudio binding can be customized to handle different devices. The Sink support is activated by default and you need no further action to use it. If you want to use another type of device, or disable the Sink type, you have to switch the corresponding binding property.
+
+-   **sink:** Allow the binding to parse sink devices from the pulseaudio server
+-   **source:** Allow the binding to parse source devices from the pulseaudio server
+-   **sinkInput:** Allow the binding to parse sink-input devices from the pulseaudio server
+-   **sourceOutput:** Allow the binding to parse source-output devices from the pulseaudio server
+
+You can use the GUI on the bindings page (click on the pulseaudio binding then "Expand for details"), or create a `<openHAB-conf>/services/pulseaudio.cfg` file and use the above options like this:
+
+```
+binding.pulseaudio:sink=true
+binding.pulseaudio:source=false
+binding.pulseaudio:sinkInput=false
+binding.pulseaudio:sourceOutput=false
```

Comment:
No need to use `org.openhab.binding.pulseaudio:` ?

Reply:
The prefix org.openhab doesn't seem to be needed in the services/pulseaudio.cfg file.
I made several tests in my dev environment, and the configuration changes are instantaneously taken with this grammar (the changes are reflected in the GUI and in the binding behavior)

Reply:
I would suggest to do it in the same way as in other bindings doc. So to be checked.

Reply:
@kaikreuzer :any recommendation? If I can avoid checking all bindings documentation,I would be happy

Reply:
For the record, I saw that the network binding documentation has a similar paragraph. I didn't check the others binding.